local/code/modules/mob/living/brain/brain_synth.dm

```diff
@@ -0,0 +1,160 @@
+/obj/item/organ/brain/synth
+	name = "compact positronic brain"
+	slot = ORGAN_SLOT_BRAIN
+	zone = BODY_ZONE_CHEST
+	organ_flags = ORGAN_ROBOTIC | ORGAN_SYNTHETIC_FROM_SPECIES
+	maxHealth = 2 * STANDARD_ORGAN_THRESHOLD
+	desc = "A cube of shining metal, four inches to a side and covered in shallow grooves. It has an IPC serial number engraved on the top. It is usually slotted into the chest of synthetic crewmembers."
+	icon = 'local/icons/obj/medical/organs/organs.dmi'
+	icon_state = "posibrain-ipc"
+	/// The last time (in ticks) a message about brain damage was sent. Don't touch.
+	var/last_message_time = 0
+	organ_traits = list(TRAIT_SILICON_EMOTES_ALLOWED)
+	var/obj/item/modular_computer/pda/synth/internal_computer
+	actions_types = list(/datum/action/item_action/synth/open_internal_computer)
+
+/obj/item/organ/brain/synth/Initialize(mapload)
+	. = ..()
+	internal_computer = new(src)
+	ADD_TRAIT(src, TRAIT_SILICON_EMOTES_ALLOWED, INNATE_TRAIT)
```

Comment:
this shouldn't be needed? the `organ_traits` list is added to the owner in `on_mob_insert` and `TRAIT_SILICON_EMOTES_ALLOWED` is only ever checked on the mob

Reply:
yep, tested